engines/mohawk/myst_stacks/myst.cpp

```diff
@@ -2292,45 +2293,76 @@ uint16 Myst::rocketSliderGetSound(uint16 pos) {
 	return (uint16)(9530 + (pos - 216) * 35.0 / 61.0);
 }
 
+
+uint16 Myst::rocketCheckSolutionLogicHelper(uint16 sound1, uint16 sound2, uint16 logictype) {
```

Comment:
You don't need to pass logictype with each call, you can just process the fuzzy_logic value inside. Also, the name of this function is not helpful. Something like ``rocketCheckIfSoundMatches()`` would be more descriptive

Reply:
Done